docs/user_guide.md

```diff
@@ -148,6 +148,53 @@ The `sly_data` dictionary can be passed along with the chat_request from the cli
 The LLMs won’t see the `sly_data`, but the coded tools can access it.
 Useful to hold onto a user id and tokens for instance.
 
+This policy is one of securtiy-by-default, whereby no sly_data gets out of the agent network
+at all unless otherwise specified. Suppose you have an agent network that takes in two numbers,
+the name of an opertion (say, addition/subtraction/multiplication/division), and asks a coded
+tool to perform the operation on the numbers. To pass the numbers as sly data to the coded tool,
+you must specify the following in the .hocon file:
```

Comment:
This is not correct.  The "to_downstream" does not affect CodedTools at all. It only effects external networks,
that is, agent networks that are called by other agent networks.